electron-app/src/views/Dashboard.vue

```diff
@@ -1,46 +1,75 @@
 <template>
   <v-container>
-    <v-row>
-      <v-col>
-        <h1 class="page-header">Dashboard</h1>
+    <v-row style="margin: 0;">
+      <v-col cols="12">
+        <overall-balances></overall-balances>
       </v-col>
     </v-row>
-
-    <div class="dashboard__information-boxes">
-      <exchange-box
-        v-for="exchange in exchanges"
-        :key="exchange.name"
-        :name="exchange.name"
-        :amount="exchange.total"
-      ></exchange-box>
-      <information-box
-        v-if="!blockchainTotal.isZero()"
-        id="blockchain_box"
-        icon="fa-hdd-o"
-        :amount="blockchainTotal"
-        blockchain
-      ></information-box>
-      <information-box
-        v-if="!fiatTotal.isZero()"
-        id="banks_box"
-        icon="fa-university"
-        :amount="fiatTotal"
-      ></information-box>
-    </div>
-
-    <v-row>
-      <v-col cols="12">
-        <h1 class="page-header">All Balances</h1>
+    <v-row class="flex" style="margin: 0; justify-content: space-evenly;">
+      <v-col cols="12" md="4" lg="4">
+        <summary-card
+          name="exchange"
+          can-refresh
+          :is-loading="exchangeIsLoading"
+          @refresh="refreshBalance($event)"
+        >
+          <exchange-box-new
+            v-for="exchange in exchanges"
+            :key="exchange.name + '_new'"
+            :name="exchange.name"
+            :amount="exchange.total"
+          ></exchange-box-new>
+        </summary-card>
+      </v-col>
+      <v-col cols="12" md="4" lg="4">
+        <summary-card
+          name="blockchain"
+          :is-loading="blockchainIsLoading"
+          can-refresh
+          @refresh="refreshBalance($event)"
+        >
+          <blockchain-balance-card-list
+            v-for="(usdValue, protocol) in blockchainTotals"
```

Comment:
An idea here would be to actually send the user to the ethereum account balances when he clicks on the ethereum logo in the blockchain balances. And the same with the Bitcoin logo.

Does not need to happen now. Perhaps can also be combined with splitting accounts & balances a bit and organizing the input of accounts per blockchain.

Reply:
Yes we were discussing that in discord too. It would make sense if all the items behave uniformly.

Reply:
I looked at being able to send someone to a page _and_ an anchor in that page and it seems it's not out of the box supported by vue-router. We will probably need to pass some sort of prop in the router link and then do something with it on `mounted()`. Is that something we want to explore/do in this PR or just take them to the general page (i.e. in this case clicking anything except the per-exchange total would take the user to the Accounts and Balances page)

Reply:
So this is not handled in this PR, can we resolve this comment?